prototype/frameworks/mcp/README.md

```diff
@@ -0,0 +1,21 @@
+# MCP - Model Context Protocol
+
+Add the correct `API_KEY`, `BASE_URL` and `MODEL_NAME` to your `.env` file. 
+
```

Comment:
For testing with ollama mistral model, Can we mention that 'API_KEY' can be any string, `BASE_URL` is `http://localhost:11434/v1` and `MODEL_NAME' is `mistral`